packages/airnode-node/test/e2e/test-api.feature.ts

```diff
@@ -2,6 +2,7 @@ import { testApi } from '../../src/workers/local-handlers';
 import { deployAirnodeAndMakeRequests, increaseTestTimeout } from '../setup/e2e';
 
 it('makes a call to test the API', async () => {
+  console.log('aaa');
```

Comment:
```suggestion
```

Reply:
I've created this issue https://api3dao.atlassian.net/browse/AN-491 since it's hard to cleanup these messages after finishing debugging.